### ISSUE TITLE
Question: Doesn't it support windows system?

### ISSUE THREAD

**Issue by nurmohammed840** [littledivy/autopilot-deno#38]
Some commend didn't work on my system...

**Comment by littledivy** [comment]
Which commands? Also please specify your Deno version.

**Comment by nurmohammed840** [comment]
```
deno 1.8.3 (release, x86_64-pc-windows-msvc)
v8 9.0.257.3
typescript 4.2.2
```
Tried exemple:
```ts
import AutoPilot from 'https://deno.land/x/autopilot@0.2.1/mod.ts';

// create a new AutoPilot instance.
var pilot = new AutoPilot();

// type a string
await pilot.type("Yay! This works"); 

// alert something
await pilot.alert("This is a alert");

// get screen size
console.log(await pilot.screenSize()); 

// move mouse
await pilot.moveMouse(200, 400);

// take a full-screen screenshot
await pilot.screenshot("screenshot.png");
```
Only `pilot.type` and `pilot.screenSize` worked.

**Comment by littledivy** [comment]
@nurmohammed840 Can you see if you can reproduce this in the latest autopilot release (`0.3.0`)?

**Comment by nurmohammed840** [comment]
I tried it out in Windows and Ubuntu 20.04 ( WSL2.0 ), 

```
deno 1.12.1 (release, x86_64-pc-windows-msvc)
v8 9.2.230.14
typescript 4.3.5
```

Unfortunately, It can't  download plugin from internet... And give this error message: 

```
error: Uncaught (in promise) Error: downloading plugin "autopilot_deno" from "https://github.com/manyuanrong/deno-plugin-prepare/releases/download//autopilot_deno.dll" failed.
    throw Error(`downloading plugin "${name}" from "${remoteUrl}" failed.`);
          ^
    at downloadFromRemote (https://deno.land/x/plugin_prepare@v0.8.0/mod.ts:81:11)
    at async download (https://deno.land/x/plugin_prepare@v0.8.0/mod.ts:51:7)
    at async prepare (https://deno.land/x/plugin_prepare@v0.8.0/mod.ts:65:21)
    at async https://deno.land/x/autopilot@0.3.0/src/bind.js:32:3
```

**Comment by littledivy** [comment]
Oops. Fixed plugin download in `0.3.2`
